mat/svd.go

```diff
@@ -147,6 +149,26 @@ func (svd *SVD) Kind() SVDKind {
 	return svd.kind
 }
 
+// Rank returns the rank of A based on the count of singular values greater than
+// rcond scaled by the first singular value.
```

Comment:
I'd say "the largest" instead of "the first".

Reply:
Done